shared/teams/team/settings-tab/retention/index.js

```diff
@@ -141,17 +145,32 @@ class RetentionPicker extends React.Component<Props, State> {
     this._init()
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  static getDerivedStateFromProps(props, state) {
```

Comment:
can we just use componentDidUpdate?

Reply:
Ok

Reply:
https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html